input/pagecontent/indv-reporting.md

```diff
@@ -92,16 +92,16 @@ The following resources are used in all individual reporting transactions.
 |---|---|---|
 |Measure|CRMI Shareable Measure Profile|[CRMI Shareable Measure]|
 |Individual MeasureReport|DEQM Individual MeasureReport Profile|[DEQM Individual MeasureReport Profile]|
-|Organization|QI Core Organization Profile|[QI Core Organization]|
-|Patient|QI Core Patient Profile|[QI Core Patient]|
+|Organization|QI-Core Organization Profile|[QI-Core Organization]|
```

Comment:
Change them to QICore instead of QI-Core, e.g., QICore Organization to match the specific profile name